internal/repository/manifest/orphan/client.go

```diff
@@ -0,0 +1,29 @@
+package orphan
```

Comment:
The repository impl does not know of the use case it is used in "orphaned manifest etc."
You can just put this repo impl to `internal/repository/kyma/client.go`, since it is actually a kyma repository implementation.